packages/kit/src/components/AccountSelector/AccountSelectorTrigger/AccountSelectorTriggerDApp.tsx

```
@@ -282,73 +281,47 @@ export function AccountSelectorTriggerBrowserSingle({ num }: { num: number }) {
-      wallet?.name,
-    ],
-  );
   return (
-    <Tooltip
-      shortcutKey={EShortcutEvents.AccountSelector}
-      renderTrigger={trigger}
-      renderContent={intl.formatMessage({ id: ETranslations.global_account })}
-      placement="bottom"
-    />
+    <XStack
+      role="button"
```

Biome

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)